lib/event_framework/event_processor_supervisor.rb

```diff
@@ -10,16 +10,15 @@ class EventProcessorSupervisor
 
     def initialize(
       processor_classes:,
-      process_manager: Forked::ProcessManager.new(logger: Logger.new(STDOUT)),
-      bookmark_repository_class: BookmarkRepository,
-      projection_database:,
-      event_source:
+      projection_database:, event_source:, tracer:, process_manager: Forked::ProcessManager.new(logger: Logger.new(STDOUT)),
```

Comment:
I think we actually want the default tracer passed in from here, and then the `EventProcessorWorker` will always have one so it won't need a default.